router/router.go

```diff
@@ -600,6 +600,9 @@ func (worker *workerT) handleWorkerDestinationJobs(ctx context.Context) {
 
 	failedUserIDsMap := make(map[string]struct{})
 	apiCallsCount := make(map[string]*destJobCountsT)
+	//Struct to hold unique users in the batch (worker.destinationJobs)
+	userToJobIDMap := make(map[string]int64)
+	routerJobResponses := make([]*RouterJobResponse, 0)
```

Comment:
Given the size of this codebase, I would suggest moving the initialization of those maps just before the `routerJobResponses` forloop. So it would be more readable.